apps/src/templates/MultipleAssignButton.jsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import Button from './Button';
+import i18n from '@cdo/locale';
+import {
+  assignToSection,
+  unassignSection,
+  sectionsForDropdown
+} from '@cdo/apps/templates/teacherDashboard/teacherSectionsRedux';
+import MultipleSectionsAssigner from '@cdo/apps/templates/MultipleSectionsAssigner';
+import {sectionForDropdownShape} from '@cdo/apps/templates/teacherDashboard/shapes';
+
+class MultipleAssignButton extends React.Component {
+  static propTypes = {
+    sectionId: PropTypes.number.isRequired,
+    sectionName: PropTypes.string,
+    courseId: PropTypes.number,
+    courseOfferingId: PropTypes.number,
+    courseVersionId: PropTypes.number,
+    scriptId: PropTypes.number,
+    assignmentName: PropTypes.string,
+    reassignConfirm: PropTypes.func,
+    isOnCoursePage: PropTypes.bool,
+    isStandAloneUnit: PropTypes.bool,
+    participantAudience: PropTypes.string,
+    // Redux
+    assignToSection: PropTypes.func.isRequired,
+    isRtl: PropTypes.bool,
+    sectionsForDropdown: PropTypes.arrayOf(sectionForDropdownShape).isRequired
+  };
+
+  state = {
+    confirmationDialogOpen: false,
```

Comment:
Looks like this state isn't used anywhere?

Reply:
That's what it looks like to me... I deleted it.  I think this was left over from something else I was looking at.